enterprise/frontend/src/embedding-sdk/test/storybook-themes.ts

```diff
@@ -1,5 +1,7 @@
-import { defineMetabaseTheme } from "embedding-sdk/components/public";
```

Comment:
This is a barrel file which imports tons of other files. This cause a side effect in storybook because those imported files will initiaze before the EE plugin does, and this will make SDK components that rely on EE plugins to behave incorrectly.